lib/ffmpegCommandService.js

```diff
@@ -112,6 +112,11 @@ class FfmpegCommandService {
             fadeOut: whippleCommandFadeEffectOptional
         });
 
+        const whippleCommandTextAlignment = Joi.object().keys({
+            horizontal: Joi.any().only([null, 'left', 'center', 'right']),
```

Comment:
What bounding box does the alignment use?

Reply:
See my explanation above. Let me know if it makes sense!